lib/tasks/db.js

```diff
@@ -69,22 +124,26 @@ module.exports = {
     descriptions: {
       'short': 'Deletes data from the database.',
       'long': [
-        'The command unseeds every existing seed.'
+        'The command tries unseeding every existing seed.'
       ]
     },
 
     task: function () {
       getMigrator('seeder', function (migrator) {
-        migrator.executed()
+        migrator.pending()
         .then(function (seeders) {
           if (seeders.length === 0) {
-            console.log('No executed seeders found.');
+            console.log('No seeders found.');
             process.exit(0);
           }
-        }).then(function () {
-          return migrator.down({to: 0});
+
+          return migrator.down({migrations: _.chain(seeders).pluck('file').reverse().value()});
```

Comment:
Is there are potentially scenario where you would seed your db with just some seeds and then unseeding all of them causes errors because you didn't apply them in the first place? And would that be a critical problem?